scripts/data_prep/convert_dataset_hf.py

```diff
@@ -232,7 +232,7 @@ def build_hf_dataset(
     else:
         if not isinstance(tokenizer, PreTrainedTokenizerBase):
             raise ValueError(
-                f'{tokenizer=} must be of type PreTrainedTokenizerBase')
+                f'{tokenizer} must be of type PreTrainedTokenizerBase')
```

Comment:
I don't think this is a typo, python format strings have some nice functionality :)
```
In [2]: f'{a=} blah'
Out[2]: 'a=1 blah'
```

Reply:
Uh, sorry I didn't know about that trick haha! One of my byod tests failed here, that's how I noticed it. Will close it.